index.html

```diff
@@ -41,7 +50,196 @@ <h1 class="hero_head">Hey there. I'm Jose Delphin.<br> <span>I'm a software deve
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      
+    </ul>
+  </section>
+
+  <header class="background_aboutme">
+    <h2 id="about-me" class="aboutme_head">About me</h2>
+    <section class="grid_container_aboutme">
+      <h3 class="aboutme_text">I can help you build a product ,feature or website. Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+        contact me.</h3>
```

Comment:
Since this is not a heading but a block of text, I think it will be better to use a `p` tag here.

Reply:
Great, it is done